$OpenBSD: patch-lib_taskbar_windows_xpm,v 1.1 2002/01/20 19:54:13 naddy Exp $
# Fix broken icon, so it looks better when used in other themes
--- lib/taskbar/windows.xpm.orig	Sun Jan 13 21:13:39 2002
+++ lib/taskbar/windows.xpm	Sun Jan 13 21:13:44 2002
@@ -1,29 +1,26 @@
 /* XPM */
-static char *windows[] = {
-/* width height num_colors chars_per_pixel */
-"    20    20        2            1",
-/* colors */
-". c #c0c0c0",
-"# c #000000",
-/* pixels */
-"....................",
-"....................",
-".##########.........",
-".##########.........",
-".#........#.........",
-".#........#.........",
-".#...##########.....",
-".#...##########.....",
-".#...#........#.....",
-".#...#........#.....",
-".#####...##########.",
-".....#...##########.",
-".....#...#........#.",
-".....#...#........#.",
-".....#####........#.",
-".........#........#.",
-".........#........#.",
-".........#........#."
-".........##########.",
-"....................",
-};
+static char * windows_xpm[] = {
+"20 20 3 1",
+" 	c None",
+".	c #000000",
+"+	c #C0C0C0",
+"                    ",
+"                    ",
+" ..........         ",
+" ..........         ",
+" .++++++++.         ",
+" .++++++++.         ",
+" .+++..........     ",
+" .+++..........     ",
+" .+++.++++++++.     ",
+" .+++.++++++++.     ",
+" .....+++.......... ",
+"     .+++.......... ",
+"     .+++.++++++++. ",
+"     .+++.++++++++. ",
+"     .....++++++++. ",
+"         .++++++++. ",
+"         .++++++++. ",
+"         .++++++++. ",
+"         .......... ",
+"                    "};
